Frontend/src/app/user-search-interface/user-search-interface.component.html

```diff
@@ -0,0 +1,81 @@
+
+<div class="container lg my-4">
+
+  <div class="row ">
+    <div class="col-lg-12 p-5 mb-4 border bg-light">
+
+      <div class="container-fluid mb-5">
+          <div class="row">
+      <h3 class=" col-lg-4 col-sm-12 ps-3">Driver License Query</h3>
+
+      <div class="form-check form-check-inline col-lg-1 col-sm-2 pt-2">
+        <input type="checkbox" class="form-check-input" name="hobbies" id="checkApi1">
+        <label class="form-check-label" for="checkMusic">API 1</label>
```

Comment:
`<label class="form-check-label" for="checkMusic">API 1</label>` - to for you should write id input that you want labled. This code you should change to `<label class="form-check-label" for="checkApi1">API 1</label>`, and not only this. Maybe, you'd better do code review.